storage/mysql/log_storage.go

```diff
@@ -160,15 +160,23 @@ func (t *readOnlyLogTX) GetActiveLogIDsWithPendingWork() ([]int64, error) {
 	return getActiveLogIDsWithPendingWork(t.tx)
 }
 
+func (m *mySQLLogStorage) hasher(treeID int64) (merkle.TreeHasher, error) {
+	// TODO: read hash algorithm from storage.
+	return merkle.Factory("RFC6962-SHA256")
```

Comment:
I'm thinking we should declare these hasher type strings as constants then IDEs can find usages of them easily.